tq/transfer_queue.go

```diff
@@ -344,14 +344,14 @@ func (q *TransferQueue) enqueueAndCollectRetriesFor(batch batch) (batch, error)
 					tracerx.Printf("tq: enqueue retry #%d for %q (size: %d)", count, tr.Oid, tr.Size)
 					next = append(next, t)
 				} else {
-					if !IsActionMissingError(err) {
-						q.errorc <- errors.Errorf("[%v] %v", tr.Name, err)
-					}
+					q.errorc <- errors.Errorf("[%v] %v", tr.Name, err)
```

Comment:
Ohh I see. Basically substituting the `a == nil` check to see if the action exists. Carry on :+1: